src/GraphQL.Tests/Introspection/SchemaIntrospectionTests.cs

```diff
@@ -22,9 +22,13 @@ public async Task validate_core_schema()
                 _.Query = SchemaIntrospection.IntrospectionQuery;
             });
 
-            var json = await new DocumentWriter(true).WriteToStringAsync(executionResult);
+            var json = await documentWriter.WriteToStringAsync(executionResult);
 
-            ShouldBe(json, IntrospectionResult.Data);
+            var expected = documentWriter is SystemTextJson.DocumentWriter
```

Comment:
Send through ClassData?

Reply:
I thought about this, but was using the same ClassData type for a few different Theory tests. I'll rework this in a way that it's provided by a class data but we still only need to maintain one list of `IDocumentWriter`s somewhere.

Reply:
Done. Also thought of a better way of dealing with the encoding differences.